x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -504,6 +507,40 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
             rowHeightsOptions={props.rowHeightsOptions}
             onColumnResize={onColumnResize}
             ref={dataGridRef}
+            renderCustomGridBody={
+              props.dynamicRowHeight
+                ? ({ visibleColumns: _visibleColumns, Cell }) => {
+                    return (
+                      <>
+                        {oldAlertsData
+                          .concat(
+                            isLoading
+                              ? Array.from({ length: pagination.pageSize - oldAlertsData.length })
+                              : []
+                          )
+                          .map((_row, rowIndex) => (
+                            <Row
+                              role="row"
+                              className={`euiDataGridRow ${
+                                rowIndex % 2 !== 0 ? 'euiDataGridRow--striped' : ''
+                              }`}
+                              key={rowIndex}
```

Comment:
maybe `key={`${rowIndex},${pagination.pageIndex}`}` like below, rowIndex will be identical when changing pages